packages/types/src/cart/workflows.ts

```diff
@@ -32,6 +33,12 @@ export interface CreateCartCreateLineItemDTO {
   metadata?: Record<string, unknown>
 }
 
+export interface UpdateLineItemInCartWorkflowInputDTO {
+  cart: CartDTO
```

Comment:
question: when doing an update of the item we need the entire DTO for which reason? I think, unless necessary, we should describe only what is expected and required

Reply:
The workflow is still missing some other modules, so I'd like to keep it as the full DTO until we know with certainty what properties are required.

Reply:
But I like the idea!